tools/metrics/run.sh

```diff
@@ -21,20 +23,21 @@ GRAFANA_DASHBOARD_FILE_PATH="./grafana/dashboards/buildbuddy.json"
 (
   open=$(which open &>/dev/null && echo "open" || echo "xdg-open")
   tries=100
-  while ! ( curl "$GRAFANA_STARTUP_URL" &>/dev/null && curl "http://localhost:9100/metrics" &>/dev/null ); do
+  while ! (curl "$GRAFANA_STARTUP_URL" &>/dev/null && curl "http://localhost:9100/metrics" &>/dev/null); do
```

Comment:
sorry for the misc whitespace changes; I'm using an autoformatter now -- use this URL to see the diff with whitespace removed https://github.com/buildbuddy-io/buildbuddy/pull/246/files?w=1